src/core/error.rs

```diff
@@ -38,8 +38,12 @@ pub enum Error {
     InvalidMutType(u8),
     MoreThanOneMemory,
     InvalidGlobalIdx(GlobalIdx),
+    InvalidFunctionIdx(FuncIdx),
     GlobalIsConst,
     RuntimeError(RuntimeError),
+    NotYetImplemented(&'static str),
```

Comment:
I'm opposing to introduce this, is there any reason to not just use `todo!()`?

Reply:
While sketching out a runner for the wasm specsuite, I ran into the `todo!`s for the sections a lot of times. I wanted to have avoid a `catch_unwind` call more due to esthetic reasons.

Reply:
I'm also against introducing this error variant. IMO it's okay to have `catch_unwind` in the test runner. Maybe we could even check if the panic message includes "not yet implemented" and display a simple statistic.